test/schema.sql

```diff
@@ -20,6 +21,7 @@ CREATE TABLE spaces (
   INDEX flagged (flagged),
   INDEX hibernated (hibernated),
   INDEX turbo (turbo),
+  INDEX turbo_expiration (turbo_expiration),
```

Comment:
We don't have index on hub's sql and prod db 🤔 we should remove here or add there

Reply:
Why remove here ? Isn't the goal to have it everywhere ?

Reply:
Not sure if we have any advantages of having an index for expiration? if it is important we should add it on hub and prod DB

Reply:
It’s to have a faster search for turbo spaces, as we will remove the `turbo` column soon, which is indexed

Reply:
ok lets add everywhere then, will create a PR?

Reply:
> It’s to have a faster search for turbo spaces, as we will remove the turbo column soon, which is indexed

Do you expect that we will have sorting or queries with turbo_expiration in future? 🤔  don't see anything important with current implementation of `turbo`

Reply:
filtering by turbo is expected yes